packages/browser-wallet-api/src/wallet-api.ts

```diff
@@ -1,79 +1,85 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable class-methods-use-this */
 import {
-    HandlerTypeEnum,
     InjectedMessageHandler,
-    Message,
-    MessageTypeEnum,
+    createEventTypeFilter,
+    EventType,
+    MessageType,
 } from '@concordium/browser-wallet-message-hub';
-import { EventEmitter } from 'eventemitter3';
-import { logger } from '@concordium/browser-wallet-message-hub/src/message-handlers/logger';
-import { PromiseInfo } from './promiseInfo';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type WalletEventHandler<T = any> = (payload: T) => void;
 
 export interface IWalletApi {
-    sendTransaction(): Promise<string>;
-    signMessage(): Promise<Message>;
-    getAccounts(): Promise<Message>;
+    addChangeAccountListener(handler: WalletEventHandler<string>): void;
+    sendTransaction(): Promise<boolean>;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    signMessage(): Promise<any>;
```

Comment:
These signatures will be changed, but I guess we'll do it, when we actually implement the features :upside_down_face:

Reply:
Yes, that was my thought as well.